StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Views/ManualEntryButton.swift

```diff
@@ -15,24 +15,38 @@ extension UIButton {
 
     static func makeManualEntryButton(appearance: PaymentSheet.Appearance) -> UIButton {
         let button = UIButton(type: .system)
-        button.titleLabel?.font = appearance.scaledFont(for: appearance.font.base.regular, style: .subheadline, maximumPointSize: 20)
-        button.tintColor = appearance.colors.primary
-
-        button.setTitle(.Localized.enter_address_manually, for: .normal)
-        button.titleLabel?.sizeToFit()
-
-        if let titleLabelHeight = button.titleLabel?.frame.size.height {
-            button.frame.size.height = titleLabelHeight * 2.25
-        }
-
-        button.backgroundColor = UIColor(dynamicProvider: { traitCollection in
-            if traitCollection.isDarkMode {
-                return appearance.colors.componentBackground
+        let font = appearance.scaledFont(for: appearance.font.base.regular, style: .subheadline, maximumPointSize: 20)
+        if #available(iOS 26.0, *), LiquidGlassDetector.isEnabled {
+            button.ios26_applyCapsuleCornerConfiguration()
+            button.ios26_applyGlassConfiguration()
+            button.frame.size.height = appearance.primaryButton.height
+            let textColor = UIColor(dynamicProvider: { traitCollection in
+                if traitCollection.isDarkMode {
+                    return appearance.colors.background.contrastingColor
+                }
+
+                return appearance.colors.primary
+            })
+            button.configuration?.attributedTitle = AttributedString(.Localized.enter_address_manually, attributes: AttributeContainer([.font: font, .foregroundColor: textColor]))
```

Comment:
Nit, move this to be the first line in the if-block since the else is updating titleLabel/tint/title/etc as the first thing in the else-block